tools/mo/openvino/tools/mo/back/MaxPool.py

```diff
@@ -40,7 +40,7 @@ def normalize_outputs(node: Node):
                                          'keep_output_port': node.has_and_set('remove_values_output')}).create_node()
             node.out_port(0).get_connection().set_destination(output.in_port(0))
 
-        if node.out_port(1).disconnected():
+        if node.has_port('out', 1) and node.out_port(1).disconnected():
```

Comment:
please use `is_out_port_connected`

Reply:
Unfortunately this routine not useful here, because ne need to check exactly port exising and port disconnecting:

`node.has_port('out', 1) and node.out_port(1).disconnected()`

But in `is_out_port_connected` we have different check - port existing and port connecting:

`self.has_port('out', idx, control_flow) and not self.out_port(idx, control_flow).disconnected()`

Of course I cann add new routine `is_out_port_disconnected` with necessary check, but I think we don't need it now.